### DISCUSSION TITLE
Riffusion Usage

### DISCUSSION THREAD

**Discussion by GeekyGhost** [pkuliyi2015/multidiffusion-upscaler-for-automatic1111#220] [Show and tell]
I've been using it with Riffusion to increase the size of Spectrograms to 8192 for consistency with diverse sound. I can now generate 1;21 of audio with a single Spectrograms. Examples! and settings lol. 

![00000-1341003363](https://github.com/pkuliyi2015/multidiffusion-upscaler-for-automatic1111/assets/111990299/0ecf7e70-8b4e-4a1b-803d-ace7384d5451)
![Screenshot 2023-05-27 062156](https://github.com/pkuliyi2015/multidiffusion-upscaler-for-automatic1111/assets/111990299/fed58cbc-26f1-404b-970c-f6448b2f0b88)
![Screenshot 2023-05-27 062215](https://github.com/pkuliyi2015/multidiffusion-upscaler-for-automatic1111/assets/111990299/48311adb-cafe-4b7e-95bb-7b1238ab7a95)

**Comment by ArnorWing** [comment]
I would love to hear your work. Have any links?

**Reply by GeekyGhost** [reply]
I have a few lol. I make a ton of stuff lol... 


https://soundcloud.com/geeky-ghost?utm_source=clipboard&utm_medium=text&utm_campaign=social_sharing

https://www.tiktok.com/@thegeekyghost?is_from_webapp=1&sender_device=pc

https://fb.watch/k-T--OO1o6/
